
---
goal_1: [0.48, 0.07, -0.25, -0.02, 0.67, -0.13, 0.73]
goal_2: [0.05, -0.37, -0.27, -0.01, 0.63, 0.03, 0.77]
goal_3: [0.25, 0.73, 0.38, 0.01, 0.03, 0.73, 0.69]
goal_4: [0.45, -0.53, 0.18, 0.05, -0.09, -0.69, 0.71]
num_goals: 4
